streaming/java/streaming-api/src/main/java/org/ray/streaming/operator/impl/SourceOperator.java

```diff
@@ -24,24 +24,20 @@ public void open(List<Collector> collectorList, RuntimeContext runtimeContext) {
     this.function.init(runtimeContext.getParallelism(), runtimeContext.getTaskIndex());
   }
 
-  public void process(Long batchId) {
+  public void run() {
     try {
-      this.sourceContext.setBatchId(batchId);
-      this.function.fetch(batchId, this.sourceContext);
+      this.function.run(this.sourceContext);
```

Comment:
Is it necessary to refactor this batch get/fetch model?
It's better to make a draft about source and sink protocol in our future plan.

Reply:
batchId won't be used in streaming, but we do need to  standardize source api. Perhaps we can do it in another PR,